### DISCUSSION TITLE
优化机器学习框架在deepin上的使用体验 问题

### DISCUSSION THREAD

**Discussion by zccrs** [linuxdeepin/developer-center#2862] [General | 常规讨论]
- [x] Upgrade cuda @Zeno-sole 
- [x] Release a blog for it @justforlxz

**Comment by tsic404** [comment]
version 11.4 has released url https://proposed-packages.deepin.com/dde-apricot/pool/main/n/nvidia-cuda-toolkit/

**Comment by justforlxz** [comment]
https://blog.justforlxz.com/2022/02/25/How-to-use-cuda-with-deepin/
